script/Deployment.s.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Why this script is needed? wouldn't it be better to broadcast the proposal mipm17 itself?